config.json

```diff
@@ -107,6 +107,16 @@
       ]
     },
     {
+      "uuid": "af5ccf14-eff2-4dc6-b1db-e209cddca62a",
```

Comment:
"New" exercises should be added the end of the list.  In this case after "collatz-conjecture"